### PR TITLE
Added Rocket to apps

### PR DESCRIPTION
**Pull request by florinvasilevilsan**
Rocket is a highly customisable modular CMS built on top of Node.js, Webpack, React, PHP, MySQL stack.
The framework exposes an API which is consumed by both the CMS and frontend of every website using Rocket.

The CMS provides multi-environment publishing enabling users usage of complete sandbox area comprised of draft and staging areas, allowing more junior team members to control content without affecting the live site.

Please find below a list of websites using Rocket:
 - https://kayo.digital/
- https://qssupportltd.com/
- https://beauty-school.co.uk/
- https://golborne-events.co.uk/
- https://carrousel.agency/
- https://seenheardconsulting.com/
- http://utilitas-solutions.co.uk/
- http://dayonedesign.co.uk/
- https://theglasswipeboardcompany.com/
- https://jflrecruit.com/
- https://rocketcms.io/
- https://visitkentoffers.co.uk/
- http://starboroughmanor.co.uk/
- https://penpartnership.com/
- https://hammerholdings.co.uk/
- https://patrickmansell.com/
- https://novia.co.uk/
- https://testudorecruit.com/
- https://darcydrainage.co.uk/
- https://motormode.com/
- https://donate.betteshangerparks.co.uk/
- https://samanthacusicklondon.com/

The list is bigger.

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -12921,6 +12921,26 @@
...
+        6
+      ],
+      "meta": {
+        "_csrf": ""
+      },
+      "cookies": {
+        "PHPSESSID": ""
+      },
+      "html": [
+        "<link href=[\"']/dist/frontend-[^\\.]+\\.css[\"'] rel=[\"']stylesheet[\"']>",
```
[the remark this answers is not in the document]
Comment (florinvasilevilsan) [reply src/apps.json]: This is now removed.

File: src/apps.json [hunk 2]
```diff
@@ -12921,6 +12921,26 @@
...
+        "<script>var onLoadList = \\[\\];</script>"
```
[the remark this answers is not in the document]
Comment (florinvasilevilsan) [reply src/apps.json]: This is now removed.

File: src/apps.json [hunk 3]
```diff
@@ -12921,6 +12921,26 @@
...
+      ],
+      "icon": "Rocket.svg",
+      "implies": ["React","PHP", "Bootstrap", "webpack", "Node.js", "MySQL", "Apache", "jQuery", "Moment.js", "Less"],
```
[the remark this answers is not in the document]
Comment (florinvasilevilsan) [reply src/apps.json]: Thanks for suggesting this. I have gone through all my implies apps, and studied the markers specified in their app definition and discovered PHP setting 'PHPSESSID'  by default. Removed this. I can certify that the remainder 'implies', won't be detected on our sites even though we are using those technologies.

File: src/apps.json [hunk 4]
```diff
@@ -12921,6 +12921,26 @@
...
+      "script": "/dist/frontend-[^\\.]+\\.js",
```
[the remark this answers is not in the document]
Comment (florinvasilevilsan) [reply src/apps.json]: This is now removed.

File: src/apps.json [hunk 5]
```diff
@@ -12921,6 +12921,18 @@
...
       "script": "/halo-(?:backend|frontend|common)/",
       "website": "https://github.com/ruibaby/halo"
+    },
+    "Rocket": {
+      "cats": [
+        1,
+        6
+      ],
+      "meta": {
+        "_csrf": ""
```
[the remark this answers is not in the document]
Comment (florinvasilevilsan) [reply src/apps.json]: This is now removed.

File: src/apps.json [hunk 6]
```diff
@@ -12921,6 +12921,18 @@
...
       "script": "/halo-(?:backend|frontend|common)/",
       "website": "https://github.com/ruibaby/halo"
+    },
+    "Rocket": {
+      "cats": [
+        1,
+        6
+      ],
+      "headers": {
+        "x-powered-by": "Rocket=https://rocketcms.io/"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"x-powered-by": "^Rocket=https://rocketcms.io/"` instead :)

Comment (florinvasilevilsan) [reply src/apps.json]: Done

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**